src/components/PatchNotes.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+
+// PatchNotes follows structure/layout of AboutPage.tsx
+const PatchNotes = () => {
+    const [isOpen, setIsOpen] = useState(true);
+
+    // show modal only on first visit    
+    useEffect(() => {
+        if (localStorage.getItem('visitedcount')) {
+            setIsOpen(false);
+        } else {
+            localStorage.setItem('visitedcount', '1');
+        }
+    }, []);
+        
+    
+    return(
+        <>
+            <Dialog fullWidth={true} onClose={(event, reason) => {
+                /* 
+                allow the user to exit the modal using their keyboard 
+                or by clicking outside the dialog 
+                */
+                if(reason == 'backdropClick' || reason == 'escapeKeyDown') {
+                setIsOpen(false);
+                }
+            }} open={isOpen}>
+                <DialogTitle>{"What's New - February 2023"}</DialogTitle>
+                <DialogContent>
+                    <DialogContentText>
+                        Features
+                        <ul>
+                            <li>Added 2023 Spring Quarter courses</li>
+                            <li>Courses will now share colors when added to Calendar</li>
+                            <li>Added this changelog!</li>
+                        </ul>
+                        Bug Fixes
+                        <ul>
+                            <li>Fixed issues with displaying GE-III courses</li>
+                            <li>Fixed courses with multiple locations appearing as independent events</li>
+                        </ul>
```

Comment:
I wonder if this can be done by loading a markdown file and converting it to basic HTML instead of having it hard coded in here. Would make changing the updates much easier. Seems like [this package](https://www.npmjs.com/package/react-markdown) could work.

Reply:
I thought the plan for the content was to hard code it initially (since it was the beginning of the month) and update it next sprint with a dynamic solution in time for March. Agree that react-markdown is the perfect tool for the job though.

Reply:
Yeah, hard-coding it initially was the plan and we can stick to it and decide whether or not we want markdown rendering later.

Reply:
My bad I must have missed that. I do think though that a dynamic solution is kinda weird because patch notes are coordinated with our commits anyways so might as well just do a commit to update the patch notes. This is assuming by dynamic solution we pull from a DB somewhere for the patch notes.